src/styles/components/_details-page.scss

```diff
@@ -37,7 +37,6 @@
     margin: auto;
     padding: 3.75rem 1.25rem;
     max-width: 100rem; // 1600px;
-    overflow: hidden;
```

Comment:
Is this where you were able to get rid of that horizontal scrollbar? When that scroll setting was on?

Reply:
I reverted back to your old implementation to fix the scrollbar. This overflow: hidden was just a relic that wasn't doing anything for us, but could be an accessibility or style issue going forward due to its pretty low specificity.